crates/shrs_core/src/theme.rs

```diff
@@ -16,6 +17,7 @@ impl Default for Theme {
             err_style: ContentStyle::new().red(),
             selection_style: ContentStyle::new().white(),
             completion_style: ContentStyle::new().red(),
+            suggestion_style: ContentStyle::new().grey(),
```

Comment:
for me at least the `white` and `grey` colors were identical, not sure how many people will actually have distinct colors for these - maybe should switch to a color that stands out more more suggestion.

Reply:
I think it should be unobtrusive, as its a suggestion